Project.toml

```diff
@@ -46,6 +47,8 @@ FFMPEG = "0.2 - 0.4"
 FixedPointNumbers = "0.6 - 0.8"
 GR = "0.69.5 - 0.71"
 Gaston = "1"
+HDF5 = "0.16"
+InspectDR = "0.4"
```

Comment:
Those weren't set before, and the enforced checks requested by @mkitti have the benefit of spotting this mistake.

Reply:
Cool